src/app/_components/FilterContainer.tsx

```diff
@@ -31,17 +38,27 @@ export function FilterContainer({ children }: LayoutProps) {
   )
 }
 
+const mobileFilterContainerStyle = {
+  1: 'w-full sm:w-64 md:w-44',
+  2: 'w-full sm:w-52 md:w-44',
+} as const
+
 FilterContainer.MobileFiltersAndResults = function MobileFiltersAndResults({
   search,
   category,
+  location,
   sort,
-}: FilterProps) {
+}: MobileFilterProps) {
+  const filtersCount = location ? 2 : 1
+  const filterStyle = mobileFilterContainerStyle[filtersCount]
```

Comment:
I really don't like this 🙈 Maybe this will help [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/888/files#r1875838345)

Reply:
Fixed by https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/903